src/emqx_mod_topic_metrics.erl

```diff
@@ -41,19 +41,19 @@
         ]).
 
 -export([ inc/2
-        , inc/3
-        , val/2
-        , metrics/1
-        , register/1
-        , unregister/1
-        , unregister_all/0
-        , is_registered/1
-        , all_registered_topics/0
-        ]).
+    , inc/3
```

Comment:
Please do not change the indentation.

I knew this Intellij issue from quite a few years ago.
Looks like it's still not fixed